### PR TITLE
fix mixnet ledger and test map address

### PR DESCRIPTION
**Pull request by jamesvan2019**

### CODE REVIEW

File: ledger/privpayouts.go [hunk 1]
```diff
@@ -28,8 +28,24 @@ var PrivGeneData = []GenesisInitPayout{
 var PrivGeneDataFromImport = []string{
 	"0,RmBKxMWg4C4EMzYowisDEGSBwmnR6tPgjLs,50000,1,0",
 	"0,RmHFARk5xmoMNUVJ6UCHFiWQML1vxwUhw1b,1254.345,1,0",
+	"0,Rm7B35PAP24GkZW1Za2gKrSiigZQ7M46KfL,1000.23456,1,0",
+	"0,RmCM99PchggcoZWkMVBWqAMHBcsn3T6VetG,2000.23456,1,0",
 }
 
+/**
```
Comment (dindinw) [inline ledger/privpayouts.go:35]: line35~48:
please unify comment style and make comments meaningful

File: params/params_mixnet.go [hunk 2]
```diff
@@ -41,6 +41,9 @@ var MixNetParams = Params{
 	ReduceMinDifficulty:  false,
 	MinDiffReductionTime: 0, // Does not apply since ReduceMinDifficulty false
 	GenerateSupported:    true,
+	UnlocksPerHeight:     10000 * 1e8, // every height 10000 MEER
```
Comment (dindinw) [inline params/params_mixnet.go:44]: where are those global configure items (line 44:46) coming from? and looks there is no where they used.  Please remove if unnecessary. I wonder if the `params` is a good palace to set the lock/unlock rules.

Comment (jamesvan2019) [reply params/params_mixnet.go:44]: It useful in ledgerpayout.go on line 196

File: ledger/privpayouts.go (hunk 1 above)
Comment (jamesvan2019) [reply ledger/privpayouts.go:35]: just a note

**Review by dindinw — APPROVED** [review]

**Review by lochjin — APPROVED** [review]

**Review by bCoder778 — APPROVED** [review]

**Pull request merged**